draft-ietf-cdni-ci-triggers-rfc8007bis.xml

```diff
@@ -1233,7 +1268,8 @@ uCDN-A                         dCDN-B                         dCDN-C
    <vspace blankLines="1"/>
    Value: Trigger state, as defined in <xref target="trigger-state"/>.
    <vspace blankLines="1"/>
-   Mandatory: dCDN MUST include trigger state in the trigger resource representtions it sends. The trigger state defaults to "pending" when a trigger is created and is optional in trigger updates sent by uCDN.
+   Mandatory: dCDN MUST include trigger state in the trigger resource representtions it sends. 
```

Comment:
This was pre-existing, but s/representtions/representations/

Reply:
already fixed in draft 16, it seems

Reply:
Nope, this is still there